### PR TITLE
[IMP] fieldservice_route: partner route link

### PR DESCRIPTION
**Pull request by chienandalu**
Ease service location routes handling.

cc @Tecnativa TT27084

### CODE REVIEW

**Review by pedrobaeza — APPROVED** [review]

File: fieldservice_route/views/res_partner_views.xml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <!-- Activar rutas desde el contacto -->
```
Comment (nikul-serpentcs) [inline fieldservice_route/views/res_partner_views.xml]: @chienandalu Convert in EN.

File: fieldservice_route/views/res_partner_views.xml [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+    <record id="fsm_partner_fields" model="ir.ui.view">
```
Comment (nikul-serpentcs) [inline fieldservice_route/views/res_partner_views.xml:4]:
```suggestion 
    <record id="fsm_partner_fields_inherit" model="ir.ui.view">
        <field name="name">res.partner.fsm.form.inherit</field>
```

**Review by nikul-serpentcs — CHANGES_REQUESTED** [review]
Nitpicking

File: fieldservice_route/views/res_partner_views.xml (hunk 2 above)
Comment (chienandalu) [reply fieldservice_route/views/res_partner_views.xml:4]: As a personal convention, I prefer to assign the same id to the inherited view, so when I search in the code or in xml_ids they show up immediately

**Comment by chienandalu** [comment]
Changes done

**Review by nikul-serpentcs — APPROVED** [review]
Code Review LGTM :+1:

**Comment by pedrobaeza** [comment]
It should be migrated to 13/14

**Pull request merged**